module/os/linux/zfs/zfs_vfsops.c

```diff
@@ -1734,7 +1734,8 @@ zfs_vget(struct super_block *sb, struct inode **ipp, fid_t *fidp)
 			VERIFY(zfsctl_root_lookup(*ipp, "snapshot", ipp,
 			    0, kcred, NULL, NULL) == 0);
 		} else {
-			igrab(*ipp);
+			/* Shouldn't be first ref, so igrab() cannot return NULL */
+			VERIFY3P(igrab(*ipp), != NULL);
```

Comment:
```suggestion
			VERIFY3P(igrab(*ipp), !=, NULL);
```

Fixes the build failure.

Reply:
Hopefully was fixed since then! :(